plugins/woocommerce-admin/client/products/fields/attribute-field/utils.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * External dependencies
+ */
+import { ProductAttribute } from '@woocommerce/data';
+
+/**
+ * Updates the position of a product attribute from the new items JSX.Element list.
+ *
+ * @param { JSX.Element[] } items list of JSX elements coming back from sortable container.
+ * @param { object } attributeKeyValues key value pair of product attributes.
+ */
+export function reorderSortableProductAttributePositions(
```

Comment:
It would be nice to have a few tests around this, but don't let it hold up this PR.  We can always tackle in a follow-up.